flask_project/campaign_manager/templates/campaign_detail.html

```diff
@@ -648,6 +648,20 @@ <h3 class="detail-campaign-name">{{ name }}</h3>
             map.fitBounds(e.target.getBounds());
         }
 
+        var legend = L.control({position: 'bottomleft'});
+        legend.onAdd = function (map) {
+            var div = L.DomUtil.create('div', 'legend');
+            div.innerHTML += '<span>Completeness</span><br/>';
+            div.innerHTML += '<i style="background: #00840d"></i> <span>100%</span><br/>';
```

Comment:
hi @ann26 
i think you need to make this automatically
let says there is variable for colours = ['#00840d','#a8ff4c','#b6ff00'] etc
and on colorCompleteness, please use this variable too